### PR TITLE
change: [M3-8607] - Change notification `tax_id_invalid` to `tax_id_verifying`

### PR DESCRIPTION
**Pull request by jaalah-akamai**
## Description 📝
Renames the `tax_id_invalid` notification to `tax_id_verifying` to represent what it actually does. The event `tax_id_invalid` remains unchanged.

## Changes  🔄
- Removed the event from the toast since we have a verifying and success toast already in place. No longer need to account for invalid since that's handled immediately during validation in the edit billing contact drawer.

In conjunction with APINext pr `7001`

## Preview 📷
No visual changes - ensure when changing tax id you see the loading icon. See recording in this PR for details https://github.com/linode/manager/pull/10928

## How to test 🧪

To test, this requires you to run DevEnv pointing to the branch for apinext PR `7001`

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Billing/BillingPanels/ContactInfoPanel/ContactInformation.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ const ContactInformation = (props: Props) => {
   const isChildUser = Boolean(profile?.user_type === 'child');
 
   const invalidTaxIdNotification = notifications?.find((notification) => {
```
Comment (bnussman-akamai) [inline packages/manager/src/features/Billing/BillingPanels/ContactInfoPanel/ContactInformation.tsx:87]: We should probably rename this const to be consistent

```suggestion
  const taxIdIsVerifyingNotification = notifications?.find((notification) => {
```

**Review by carrillo-erik — APPROVED** [review]

**Review by cpathipa — APPROVED** [review]

File: packages/api-v4/src/account/types.ts [hunk 2]
```diff
@@ -265,7 +265,7 @@ export type NotificationType =
   | 'user_email_bounce'
   | 'volume_migration_scheduled'
   | 'volume_migration_imminent'
-  | 'tax_id_invalid';
+  | 'tax_id_verifying';
```
[the remark this answers is not in the document]
Comment (jphilipp-akamai) [reply packages/api-v4/src/account/types.ts:268]: This has been updated and merged on the api-v4 side.

**Comment by jaalah-akamai** [comment]
@jdamore-linode Any idea why I'm getting a failure for `plan-selection.spec.ts` when using `yarn build && yarn start:manager:ci`. Hopefully it's not more gecko related issues with regionSelect

**Comment by jaalah-akamai** [comment]
Screenshot 2024-09-30 at 11 03 15 PM

**Comment by jdamore-linode** [comment]
@jaalah-akamai Mind merging in the latest from `develop` to see if that clears up the test issues? These look like test failures that might've been fixed with PRs #10976 and #10978

**Pull request merged**